ansible/roles/keystone/tasks/init_fernet.yml

```diff
@@ -1,4 +1,26 @@
 ---
+
+- name: Get info on fernet keys volume
+  shell: docker volume inspect keystone_fernet_tokens
+  register: result
+
+- name: Get Keystone fernet volume data
+  set_fact:
+    dataresult: "{{ result.stdout | from_json }}"
+
+- name: Get Keystone fernet volume mount point
+  set_fact:
+    keystone_fernet_tokens_mountpoint: "{{ dataresult[0].Mountpoint }}"
+
+- name: Create fernet token file(s)
+  become: True
+  copy:
+    dest: "{{ keystone_fernet_tokens_mountpoint }}/{{index}}"
+    content: "{{item}}"
+  loop: "{{ keystone_fernet_token_list }}"
+  loop_control:
+    index_var: index
```

Comment:
What is the default value of `index_var`? Is it not `index`?

Reply:
Evidently not. I had to add that declaration explicitly. 🤷‍♂️